supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -112,6 +112,7 @@ Ensure that user-replaced values have the following characteristics:
 * Lowercase, unless the rest of the related text is uppercase or another capitalization scheme
 * Italicized
 * If the user-replaced value is referencing a value in code or in a command that is normally monospace, also use monospace for the user-replaced value
+* If the user-replaced value appears in example output, only format the value in the output if the replaceable value was also in the command or if formatting adds value to the customer. Otherwise, leave the replaceable value unformatted as an example.
```

Comment:
Right now this reads to me like the options are to format it or not format it. But I think if you're using a replaceable value in output, it still needs formatted properly.

Maybe we could reword to be clearer that the options are to either use a replaceable value (and italicize / put it in angle brackets) or use an example value (and don't format it any special way).

```suggestion
* If you want to use a user-replaced value in example output, format the replaceable value with italics and in angle brackets. Alternatively, if you choose to use an example value instead, do not italicize the example value or place it in angle brackets.
```

Reply:
+1 to the suggested edit from @bergerhoffer. They distinguish the two alternatives more clearly.